### PR TITLE
task22

### PR DESCRIPTION
**Pull request by FedorovD**

### CODE REVIEW

File: 1-select-tasks/22.txt [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+JOIN [Orders] [o] ON [o].[CustomerId] = [c].[CustomerId]
+JOIN [Order Details] [od] ON [o].[OrderId] = [od].[OrderId]
+JOIN [Products] [p] ON [od].[ProductId] = [p].[ProductId]
+WHERE [od].[UnitPrice] = (
+
+SELECT TOP 1 MAX([odd].[UnitPrice])
+FROM [Customers] [cc]
+JOIN [Orders] [oo] ON [oo].[CustomerId] = [cc].[CustomerId]
+JOIN [Order Details] [odd] ON [oo].[OrderId] = [odd].[OrderId]
+WHERE [cc].[CompanyName] = [c].[CompanyName]
```
Comment (AlexMokin) [inline 1-select-tasks/22.txt]: it is better to use customerid

File: 1-select-tasks/22.txt [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+GROUP BY [cc].[CompanyName], [odd].[UnitPrice]
```
Comment (AlexMokin) [inline 1-select-tasks/22.txt]: why do you group by price?

Comment (FedorovD) [reply 1-select-tasks/22.txt]: I was getting the top 1 unit price in descending order (order by) and without group by it didn't work but I reconsidered my previous task solution and redoing it

**Comment by AlexMokin** [comment]
well done!

**Pull request closed without merging**